detectron2/data/build.py

```diff
@@ -103,6 +103,13 @@ def load_proposals_into_dataset(dataset_dicts, proposal_file):
     """
     Load precomputed object proposals into the dataset.
 
+    The proposal file should be a pickled dict[str->list] with the following
```

Comment:
```suggestion
    The proposal file should be a pickled dict with the following
```
bbox_mode is not a list